File: csharp/gildedrose_starter/GildedRose/GildedRoseTest.cs
using System.Collections.Generic;
using NUnit.Framework;


namespace GildedRose
{
    [TestFixture]
    public class GildedRoseTest
    {
        [Test]
        public void this_test_needs_a_better_name()
        {
            // Arrange
            IList<Item> items = new List<Item> { new Item { Name = "foo", SellIn = 0, Quality = 0 } };
            GildedRose sut = new GildedRose(items);
            
            // Act
            sut.UpdateQuality();
            
            // Assert
            Assert.AreEqual("fixme", items[0].Name);
        }
    }
}

File: java/gildedrose/src/test/java/com/gildedrose/GildedRoseTest.java
package com.gildedrose;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class GildedRoseTest {

    @Test
    public void this_test_needs_a_better_name() {
        // Arrange
        Item[] items = new Item[] { new Item("foo", 0, 0) };
        GildedRose sut = new GildedRose(items);

        // Act
        sut.updateQuality();

        // Assert
        assertEquals("fixme", sut.items[0].name);
    }
}
